cmd.sh

```diff
@@ -14,8 +14,17 @@ set -x
 # this is to prevent accidental uploads
 echo "Getting anaconda token from github secrets..."
 
-ANACONDA_ORG="scientific-python-nightly-wheels"
+ANACONDA_ORG="${INPUT_ANACONDA_NIGHTLY_UPLOAD_ORGANIZATION}"
 ANACONDA_TOKEN="${INPUT_ANACONDA_NIGHTLY_UPLOAD_TOKEN}"
+ANACONDA_LABELS="${INPUT_ANACONDA_NIGHTLY_UPLOAD_LABELS}"
+
+# if the ANACONDA_ORG is empty, exit with status -1
+# this is to prevent attempt to upload to the wrong anaconda channel
+if [ -z "${ANACONDA_ORG}" ]; then
+  echo "ANACONDA_ORG is empty , exiting..."
```

Comment:
```suggestion
  echo "ANACONDA_ORG is empty, exiting..."
```

I know that there is an extra accidental space above, but that's a typo we don't need to recreate. (this is super nitpicky, sorry, which is why I'm adding it as a suggested fix so you don't have to do anything)